src/Umbraco.Web.UI.Client/src/views/components/umb-node-preview.html

```diff
@@ -0,0 +1,12 @@
+<div class="umb-node-preview" ng-class="{'umb-node-preview--sortable': sortable, 'umb-node-preview--unpublished': published === false }">
```

Comment:
Now seeing this component view then I would highly recommend we document the JS directive code mentioned further up as I would imagine other property editor developers using this new view/component in their own stuff @madsrasmussen